requirements/test.in

```diff
@@ -5,11 +5,11 @@
 
 coverage
 ddt
-edx-lint
+edx-lint==1.3.0
```

Comment:
Restrictions like this should go in `requirements/constraints.txt` along with a comment explaining why the version was pinned.